phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixMetricsIT.java

```diff
@@ -508,11 +534,17 @@ public void testMetricsForUpsert() throws Exception {
                     assertEquals("Zero failed mutations expected", 0, metricValue);
                     mutationBatchFailedPresent = true;
                 }
+                else if (metricType.equals(MetricType.MUTATION_BATCH_SUCCESS_COUNTER)) {
+                    assertTrue("Mutation batch success count should be greater than zero",
+                            metricValue > 0);
```

Comment:
You can do a much tighter check here by checking for the exact value of mutation batches.